knowledge-base/attaching-badge-to-telerik-tabview-header-tab.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Attaching Badge to Telerik TabView Header/Tab
+description: Learn how to attach a badge to each tab in the Telerik TabView control to display additional info.
+type: how-to
+page_title: How to Attach Badge to Telerik TabView Header/Tab
+slug: attaching-badge-to-telerik-tabview-header-tab
+tags: badge, tabview, header, tab, customization
+res_type: kb
+---
+
+## Environment
+| Property | Value |
+| --- | --- |
+| Product | BadgeView for .NET MAUI |
+| Version | 6.7.0 |
+
+## Description
+I want to attach a badge to each tab in the Telerik TabView control to display the number of new items in each tab.
+
+## Solution
+Follow these steps to attach a badge to each tab in the Telerik TabView control:
+
+1. Customize the TabItem ControlTemplate by using the `HeaderItemTemplate` property of the TabView control. In the custom ControlTemplate, add a BadgeView control, attached to a Label or any other suitable control within the TabItem.
```

Comment:
When mentioning ControlTemplate, use either `ControlTemplate` in monospace or "control template" as two words.
```suggestion
1. Customize the `TabItem` control template by using the `HeaderItemTemplate` property of the TabView control. In the custom control template, add a BadgeView control, attached to a Label or any other suitable control within the TabItem.
```